src/GitHub.Exports/Services/Services.cs

```diff
@@ -120,5 +121,28 @@ public static string GetFileNameFromActiveDocument()
             var fullName = Dte2?.ActiveDocument?.FullName;
             return Path.GetFileName(fullName);
         }
+
+        /// <summary>
+        /// There doesn't appear to be a command that directly opens a target repo.
+        /// Our workaround is to create, open and delete a solution in the repo directory.
+        /// This triggers an event that causes the target repo to open. ;)
+        /// </summary>
+        /// <param name="directory">The target repo directory. </param>
+        public static void OpenRepository(string directory)
```

Comment:
Weeell, the caller doesn't need to know the magic 😉 If it lives in the `VSServices` class, then it should (hopefully) be fairly obvious that the intent is to open a repository in the context of VS (and not, say, for doing git operations). Probably should have some docs on it too.

Might be a good idea to call this `TryOpenRepository` though, because as I mention in another comment, if this fails then the caller is going to need to know that, in order to either fallback to an alternate codepath, or report to the user that the action failed (and update/not update the UI accordingly).  The user doesn't need to know that we failed to delete a fake solution or whatever failed in the method, they just need to know that the action they were expecting to have happen didn't happen, and guide the user on how they can do it in a different way.